probe-rs-tools/src/bin/probe-rs/cmd/complete.rs

```diff
@@ -291,6 +308,12 @@ impl ShellExt for PowerShell {
 }
 
 fn write_script(path: &Path, script: &str) -> Result<()> {
+    if let Some(parent) = path.parent() {
+        if !parent.exists() {
+            std::fs::create_dir_all(parent).context("Failed to create directory")?;
```

Comment:
Can we also print the script if this fails so the user can install it manually? :)